src/app/(proper_react)/redesign/(authenticated)/admin/feature-flags/AddFeatureFlag.tsx

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+export const AddFeatureFlag = () => {
+  // @ts-ignore FIXME determine correct event type
+  const handleSubmit = void (async (event) => {
+    event.preventDefault();
+
+    const data = {
+      name: event?.target?.name.value,
+      isEnabled: event?.target?.isEnabled.value ? true : false,
+      dependencies: event?.target?.dependencies.value.split(","),
+      allowList: event?.target?.allowList.value.split(","),
+      waitList: event?.target?.waitList.value.split(","),
+      owner: event?.target?.owner.value,
+    };
+
+    const endpoint = "/api/v1/admin/feature-flags" as string;
+
+    const options = {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(data),
+    };
+
+    // Send the form data to our forms API on Vercel and get a response.
```

Comment:
Wait, are we using Vercel? I thought we were using GCP and Netlify.

Reply:
No, sorry this is just from a tutorial, I'll remove it. We're using GCP and k8s :)